demo/js/components/accessibility-demo.js

```diff
@@ -1,45 +1,110 @@
 import React from "react";
-import { VictoryChart } from "Packages/victory-chart/src/index";
-import { VictoryBar } from "Packages/victory-bar/src/index";
+import { VictoryBar } from "Packages/victory-bar";
+import { VictoryChart } from "Packages/victory-chart";
+import { VictoryBoxPlot } from "Packages/victory-box-plot";
+import { LineSegment, Whisker, Border } from "victory-core/src";
 
-export default class App extends React.Component {
-  render() {
-    const containerStyle = {
-      display: "flex",
-      flexDirection: "row",
-      flexWrap: "wrap",
-      alignItems: "center",
-      justifyContent: "center",
-      maxWidth: "40%",
-      margin: "3%"
-    };
+const containerStyle = {
+  display: "flex",
+  flexDirection: "row",
+  flexWrap: "wrap",
+  alignItems: "center",
+  justifyContent: "flex-start"
+};
 
-    const chartContainerStyle = {
-      display: "flex",
-      flexDirection: "column",
-      alignItems: "center",
-      width: "100%",
-      height: "100%"
-    };
+const chartContainerStyle = {
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+  width: "50%",
+  height: "50%"
+};
 
+export default class App extends React.Component {
+  render() {
     return (
-      <div className="demo" style={containerStyle}>
-        <div style={chartContainerStyle}>
-          <h3>Tabbable with aria-labels: bar chart</h3>
-          <VictoryChart domainPadding={{ x: 40, y: 40 }}>
-            <VictoryBar
-              data={[
-                { x: "A", y: 1 },
-                { x: "B", y: 3 },
-                { x: "C", y: 5 },
-                { x: "D", y: 7 }
-              ]}
-              ariaLabel={({ datum }) => `bar-value-${datum.x}`}
-              tabIndex={({ index }) => index + 1}
-            />
-          </VictoryChart>
+      <>
+        <div>
+          <h2>Tabbable charts with aria-labels</h2>
+        </div>
+        <div className="demo" style={containerStyle}>
+          {/** BAR CHART */}
+          <div style={chartContainerStyle}>
+            <h3> Bar chart</h3>
+            <VictoryChart domainPadding={{ x: 40, y: 40 }}>
+              <VictoryBar
+                data={[
+                  { x: "A", y: 1 },
+                  { x: "B", y: 3 },
+                  { x: "C", y: 5 },
+                  { x: "D", y: 7 }
+                ]}
+                ariaLabel={({ datum }) => `bar-value-${datum.x}`}
```

Comment:
`ariaLabel` and `tabIndex` should be on the `Bar` `dataComponent` in this example.